### PR TITLE
PXB-2848 Document how to restore a single table from a full backup

### PR DESCRIPTION
**Pull request by patrickbirch**
a On branch pxb-2848
	new file:   docs/restore-single-table.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/restore-single-table.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+2. Back up the table:
+
+    Now, let's back up just the mytable table. Run the `xtrabackup` command with the `--tables` option to specify the table and other connection details.
+
+    ```{.bash data-prompt="$"}
+    $ ./xtrabackup --user=backup --password=password --backup --tables=mytable --target-dir=$HOME/dbbackup -S $HOME/test/socket.sock --datadir=$HOME/test/data
+    ```
+
+    XtraBackup copies the `mytable.ibd` file and other required files to the backup directory.
+
```
Comment (satya-bodapati) [inline docs/restore-single-table.md:56]: This doesn't copy the schema of the table. I think we should show a way to backup the schema of the table and use that schema to create table/ discard, then import.

File: docs/restore-single-table.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+| Potential Data Inconsistency | If the table you restore depends on other tables (like foreign keys or constraints), you might face data consistency issues. You need to ensure that related data remains consistent.                  |
+| Manual Effort            | The process involves several manual steps, such as creating a dummy table, discarding the tablespace, and importing the tablespace. This requires careful execution to avoid mistakes.                 |
+
+The following procedure walks you through backing up and restoring a single table.
+
+1. Verify the table checksum
+
+    Before starting, let's confirm the current state of `mytable`. Use the `CHECKSUM TABLE` command to get a checksum value. This value represents a fingerprint of the table's contents. We'll use this command again after the restore to verify success.
+
+    ```{.bash data-prompt="mysql>"}
```
Comment (satya-bodapati) [inline docs/restore-single-table.md:33]: This assumes no concurrent writes to table. Only then we can use checksum table.

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: docs/restore-single-table.md (hunk 1 above)
Comment (patrickbirch) [reply docs/restore-single-table.md:56]: Added a Partial backup section with mysqldump

File: docs/restore-single-table.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+    ```
+
+    * Replace `<directory>` with the path containing the table files (usually inside the data directory) for the specific schema you want to back up. 
+    * Replace `outdir` with the directory where you want to store the partial XtraBackup.
+
+4. (Optional) Resume writes
+
+    If you stopped writes earlier, run `mysqladmin UNLOCK TABLES` to unlock the tables and resume normal operations.
+
+Restore the Backup
```
Comment (alina-derkach-oaza) [inline docs/restore-single-table.md]: Should Restore the Backup be a heading?

**Pull request closed without merging**